src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/CustomMetadataFields.scala

```diff
@@ -32,8 +32,7 @@ object CustomMetadataFields extends DataTypeFields {
       values: List[CustomMetadataValues],
```

Comment:
Not related to your changes but an unused import: `import uk.gov.nationalarchives.tdr.api.graphql.fields.DataTypeFields`